CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Icons Changelog
```

Comment:
There's an `Updating the Icon Font` section in the `README` containing steps to release that should include one for this part

Reply:
done